controllers/login.js

```diff
@@ -213,12 +215,19 @@ router.get('/login/systems/:schoolId', (req, res, next) => {
 });
 
 router.get('/logout/', (req, res, next) => {
-	api(req).del('/authentication') // async, ignore result
-		.catch((err) => { logger.error('error during logout.', { error: err.toString() }); });
-	return authHelper
-		.clearCookie(req, res, { destroySession: true })
-		.then(() => res.redirect('/'))
-		.catch(next);
+	if (res.locals.csrfToken) {
```

Comment:
But I think, this will be always fulfilled. We need to add the csrf token to the request and check it, like here:
![image](https://user-images.githubusercontent.com/65842673/95414393-15927f00-092e-11eb-837a-c43991bcef7c.png)

Reply:
But maybe I didn't get, how this csrf token is created and handled ...